src/workloads/execution/FilterWithLogicalExpressions.yml

```
@@ -3,12 +3,30 @@ Owner: "@mongodb/query"
 Description: >
   This workload stresses the query execution engine by running queries with complex logical
   expressions that never match a document in the collection.
+  Each workload name consists of several parts: '{SyntaxType}{PredicateType}'.
+  'SyntaxType' can be:
+    - 'AggregationExpressions' means expressions which can be used inside $expr
+    - 'MatchExpressions' means operators of the find command language
```

```suggestion
    - 'AggregationExpression' means expressions which can be used inside $expr
    - 'MatchExpression' means operators of the find command language
```

This is a total nitpick, but if you make these names singular rather than plural the names used below will all roll off the tongue better. For example, `AggregationExpressionWidePredicate` sounds better to me than `AggregationExpressionsWidePredicate`.